### PR TITLE
chore: appease the linter (3/?), the big gofumpt 

### PR DESCRIPTION
**Pull request by rbtr**
**Reason for Change**:

Correct gofumpt findings, automatically, by `$ gofumpt -w -s .`

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

**Comment by rbtr** [comment]
@vakalapa adding you as reviewer because github says you most recently edited all these files 😛

File: proto/nodenetworkservice/3.302.0.744/NodeNetworkService.pb.go [hunk 1]
```diff
@@ -627,19 +628,22 @@ func file_NodeNetworkService_proto_rawDescGZIP() []byte {
...
-	(*ConfigureNetworkingRequest)(nil),           // 1: azure.nodenetworkservice.ConfigureNetworkingRequest
-	(*ConfigureNetworkingResponse)(nil),          // 2: azure.nodenetworkservice.ConfigureNetworkingResponse
-	(*PingNodeNetworkServiceRequest)(nil),        // 3: azure.nodenetworkservice.PingNodeNetworkServiceRequest
-	(*PingNodeNetworkServiceResponse)(nil),       // 4: azure.nodenetworkservice.PingNodeNetworkServiceResponse
-	(*ConfigureContainerNetworkingRequest)(nil),  // 5: azure.nodenetworkservice.ConfigureContainerNetworkingRequest
-	(*ConfigureContainerNetworkingResponse)(nil), // 6: azure.nodenetworkservice.ConfigureContainerNetworkingResponse
-	(*ContainerIPAddress)(nil),                   // 7: azure.nodenetworkservice.ContainerIPAddress
-	(*ContainerNetworkInterface)(nil),            // 8: azure.nodenetworkservice.ContainerNetworkInterface
-}
+var (
```
Comment (vakalapa) [inline proto/nodenetworkservice/3.302.0.744/NodeNetworkService.pb.go:631]: Aren't .pb.go files auto generated ? These changes can get overwritten on any new updates to the proto files right ?

File: cnm/plugin.go [hunk 2]
```diff
@@ -85,12 +85,12 @@ func (plugin *Plugin) EnableDiscovery() error {
 
 	// Create the spec directory.
 	path := plugin.getSpecPath()
-	os.MkdirAll(path, 0755)
+	os.MkdirAll(path, 0o755)
```
Comment (vakalapa) [inline cnm/plugin.go:88]: I am inclined towards checking errors on mkdirall. wdyt?

File: cns/ipamclient/ipamclient_test.go [hunk 3]
```diff
@@ -14,13 +14,14 @@ import (
 	"github.com/Azure/azure-container-networking/common"
 )
 
-var mux *http.ServeMux
-var ipamQueryUrl = "localhost:42424"
-var ic *IpamClient
+var (
+	mux          *http.ServeMux
```
Comment (vakalapa) [inline cns/ipamclient/ipamclient_test.go:18]: Looks like "mux" is not used, can we check with whoever added if this can be removed ?

**Review by vakalapa — COMMENTED** [review]
Minor comments, almost all changes seem straight forward.

File: proto/nodenetworkservice/3.302.0.744/NodeNetworkService.pb.go (hunk 1 above)
Comment (rbtr) [reply proto/nodenetworkservice/3.302.0.744/NodeNetworkService.pb.go:631]: dropped those from this change, good call

File: telemetry/telemetry_test.go [hunk 4]
```diff
@@ -22,10 +22,12 @@ const (
...
 )
 
-var reportManager *ReportManager
-var tb *TelemetryBuffer
-var ipamQueryUrl = "localhost:3501"
-var hostAgentUrl = "localhost:3500"
+var (
+	reportManager *ReportManager
+	tb            *TelemetryBuffer
+	ipamQueryUrl  = "localhost:3501"
```
[the remark this answers is not in the document]
Comment (rbtr) [reply telemetry/telemetry_test.go:28]: i would rather keep this scoped to just the automated gofumpt. there are _lots_ of lint fixes yet to come

File: cnm/plugin.go (hunk 2 above)
Comment (rbtr) [reply cnm/plugin.go:88]: i agree, but not in this PR

File: common/plugin.go [hunk 5]
```diff
@@ -30,8 +30,7 @@ type NetApi interface {
 }
 
 // IPAM internal interface.
-type IpamApi interface {
-}
+type IpamApi interface{}
```
[the remark this answers is not in the document]
Comment (rbtr) [reply common/plugin.go:33]: not in this PR

**Review by vakalapa — APPROVED** [review]
🚀

**Review by vakalapa — APPROVED** [review]
Looks good, only 3 files got updated.

**Comment by rbtr** [comment]
bypassing lint check since it is just a big false positive

**Pull request merged**